.gitignore

```diff
@@ -1,5 +1,5 @@
 public
-resources
+resources/_gen
```

Comment:
This ensures that the `resources` files aren't committed while preventing the `layouts/resources` files from being ignored.